vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -257,10 +259,26 @@ def create_weights(
             else:
                 layer.register_parameter("input_scale", None)
 
+    def dequant_block_fp8_weight(self, layer) -> torch.Tensor:
```

Comment:
This is added by INC team, @yiliu30 , please check if it is necessary to keep it in the fp8,py?